package.json

```diff
@@ -26,15 +26,15 @@
   "scripts": {
     "tslint": "tslint -c tslint.json 'src/**/*.ts' -p ./ --fix",
     "clean": "rm -rf ./lib",
-    "compile": "tsc",
+    "tsc": "tsc",
     "prettier": "prettier --write 'src/**/*.ts'; prettier --write 'test/**/*.ts'",
-    "prepare": "npm run clean && npm run prettier && npm run tslint && npm run compile",
+    "compile": "npm run clean && npm run prettier && npm run tslint && npm run tsc",
```

Comment:
revert to prepare. it's called automatically during npm run install. this will break travis

Reply:
I know I suggest updating travis and maintain same terminology to all of our repositories

Reply:
this is not related to travis but to the npm workflow. consider it as a preserved word